src/components/AddToPlaylist.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <VDialog v-model="dialog" width="600px" v-if="playlistOptions.length">
+    <template v-slot:activator="{ on }">
+      <VBtn
+        v-on="on"
+        @click.stop.prevent
+        class="actions__button"
+        color="primary"
+        text
+        icon
+        small
+      >
+        <VIcon>mdi-playlist-plus</VIcon>
+      </VBtn>
+    </template>
+    <VCard>
+      <VCardTitle>
+        <span class="text-h5">{{
+          $t("music.playlist.add-item", { obj: item.name || item.title })
+        }}</span>
+      </VCardTitle>
+      <VCardText>
+        <VContainer>
+          <VRow>
+            <VCol cols="12">
+              <VCombobox
+                :items="editablePlaylists"
+                :filter="filterName"
+                cache-items
+                item-text="name"
+                item-value="id"
+                :label="$tc('music.playlists', 1)"
+                return-object
+                v-model="selectedPlaylist"
+              />
+            </VCol>
+          </VRow>
+        </VContainer>
+      </VCardText>
+      <VDivider></VDivider>
+      <VCardActions>
+        <VSpacer></VSpacer>
+        <VBtn
+          color="primary"
+          class="ma-2"
+          type="submit"
+          :disabled="selectedPlaylist === null"
+          @click="addItemToPlaylist"
+        >
+          {{ $t("music.playlist.add") }}
+        </VBtn>
+      </VCardActions>
+    </VCard>
+  </VDialog>
+</template>
+
+<script>
+import { mapActions, mapGetters } from "vuex";
+
+export default {
+  name: "AddToPlaylistDialog",
+  props: {
+    item: {
+      type: Object,
+      required: true,
+    },
+    type: {
+      type: String,
+      required: true,
+    },
+  },
+  data() {
+    return {
+      dialog: false,
+      selectedPlaylist: null,
+    };
+  },
+  computed: {
+    ...mapGetters("auth", ["currentUser"]),
+    ...mapGetters("playlists", ["editablePlaylists"]),
+    playlistOptions() {
+      return this.editablePlaylists.filter(
+        (p) => p.playlist_type === this.type
```

Comment:
This doesn't seem to work.

Reply:
I just wasn't using this computed prop 🤦‍♂️